blockly-core/core/ui/fields/field_label.js

```diff
@@ -74,7 +74,9 @@ Blockly.FieldLabel.prototype.init = function(block) {
 
   // Configure the field to be transparent with respect to tooltips.
   this.textElement_.tooltip = this.sourceBlock_;
-  Blockly.Tooltip && Blockly.Tooltip.bindMouseEvents(this.textElement_);
+  if (!this.sourceBlock_.blockSpace.blockSpaceEditor.disableTooltip) {
```

Comment:
Only concern with this chaining is possible undefined properties along the way—i.e., if there were blocks with labels that exist outside of a blockSpaceEditor context. But `BlockSpace`s take in an editor in constructor, so should be fine, and just tested out mousing over a block within the "if else" star-button mutator blockspace and appeared not to cause any sort of exception :+1:.